councilmatic_core/models.py

```diff
@@ -309,8 +309,16 @@ class Meta:
 
 class EventManager(models.Manager):
     def get_queryset(self):
-        return super().get_queryset().annotate(start_time=Cast('start_date',
-                                                               models.DateTimeField()))
+        return super().get_queryset().annotate(
+            start_time=Cast(
+                Case(
+                    When(start_date='', then=None),
+                    default='start_date',
+                    output_field=models.CharField()
```

Comment:
At first I thought that we might be able to set `output_field` to `models.DateTimeField()` and skip the enclosing `Cast` call altogether. Unfortunately, that doesn't work! Not entirely sure why, since `output_field` isn't really documented properly.

Reply:
i've had trouble with that, too. boo!